### ISSUE TITLE
Guard against idle mode only methods?

### ISSUE THREAD

**Issue by caternuson** [adafruit/Adafruit_CircuitPython_SCD4X#10]
Some commands only work in idle mode, altitude for example:
![image](https://user-images.githubusercontent.com/8755041/146452301-b2faf425-b7cf-4260-9a2d-1dd351763f41.png)

This can lead to potentially unexpected hiccups in user code:
```python
>>> scd4x.altitude = 42
>>> scd4x.altitude
42
>>> scd4x.start_periodic_measurement()
>>> scd4x.altitude
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "adafruit_scd4x.py", line 284, in altitude
  File "adafruit_scd4x.py", line 307, in _send_command
OSError: [Errno 5] Input/output error
>>> 
```

Could just catch this in user code, but might be better to handle this more gracefully here in driver code?

**Comment by ladyada** [comment]
ermm maybe throw an exception so at least they know what happened?

**Comment by caternuson** [comment]
like just catch the `OSError` here and throw something else with a more helpful message?

**Comment by ladyada** [comment]
yaz

**Comment by caternuson** [comment]
Fixed with #13.
